README.md

```diff
@@ -35,7 +35,7 @@ repositories are:
 ## Requirements
 
 - Ruby 2.1.3
-- PostgreSQL 9.3+
+- PostgreSQL 9.2
```

Comment:
It might be worth changing this to `9.2+`.  9.3 does work, as that's what I've been using.